packages/nodes-base/nodes/Freshdesk/GenericFunctions.ts

```diff
@@ -42,7 +41,21 @@ export async function freshdeskApiRequest(
 	try {
 		return await this.helpers.request!(options);
 	} catch (error) {
-		throw new NodeApiError(this.getNode(), error);
+		const errorData = (error.message || '').split(' - ')[1] as string;
+		if (errorData) {
+			const parsedError = JSON.parse(errorData.trim());
+			console.log(parsedError);
+			let { message } = parsedError;
+			const { errors, description } = parsedError;
+			if (!message) {
+				message =
+					((errors as IDataObject[]) || [])
+						.map((error: IDataObject) => error.message as string)
+						.join(', ') || error.message;
+			}
+			throw new NodeApiError(this.getNode(), error, { message, description });
+		}
+		throw new NodeApiError(this.getNode(), error, { message: error.message });
```

Comment:
if we only pass the error object, does not the error handler extract the error message for you?

Reply:
strange, it was giving me "Unknown error, check detail info below", now it's parsing errors correctly